docs/spec.md

```diff
@@ -5054,7 +5054,7 @@ a `result` tensor.
 The following diagram shows how elements in `result` are computed from
 `operand` and `source` using a concrete example.
 
-![](images/spec/select_and_scatter.svg)
+![select_and_scatter](images/spec/select_and_scatter.svg)
```

Comment:
I prefer the default description to be the same as file name like here (but up to you). Could you also update others (or update this one to append `example`) for consistency? (e.g.
`![scatter example](images/spec/scatter.svg)` to `![scatter](images/spec/scatter.svg)`)

Reply:
Good catch, changed all to match the filename.